tests/test_pdf_extract.py

```diff
@@ -6,14 +7,13 @@
 class MyTestCase(unittest.TestCase):
     def test_get_pdf_objects(self):
         config = {
-            "teams": {
-                "Mock Team": {
-                    "team_dashboards": {"Mock Team Dashboard": {"Filename": "test.pdf"}}
+            "Mock Team": {
+                "team_dashboards": {
+                    "Mock Team Dashboard": {"Filename": "sample_report.pdf"}
                 }
-            },
-            "config": {"Download_path": "tests/fixtures/"},
+            }
         }
-        result = get_pdf_objects(config)
+        result = get_pdf_objects(config, pathlib.Path("tests/fixtures/"))
```

Comment:
How about adding path using `__file__` so that the tests can be run from any directory.